package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "sw-precache",
-  "version": "4.2.2",
+  "version": "4.2.3",
```

Comment:
Generally suggest not bumping package versions in feature PRs, but you know what you're doing :)